rust/connlib/tunnel/src/tests/stub_portal.rs

```diff
@@ -75,6 +88,10 @@ impl StubPortal {
                     .cloned()
                     .map(client::ResourceDescription::Dns),
             )
+            // TODO: Enable once we actually implement the Internet resource
+            // .chain(iter::once(client::ResourceDescription::Internet(
+            //     self.internet_resource.clone(),
+            // )))
```

Comment:
@conectado In the PR where you add the Internet Resource, enabling this is all you should need to be doing.

Reply:
I'd suggest we merge this as is and implement any potential fixes either in the implementation PR or separately.